boards/catie/zest_core_stm32l4a6rg/sixtron_connector.dtsi

```diff
@@ -4,6 +4,7 @@
  */
 
 #include <zephyr/dt-bindings/gpio/sixtron-header.h>
+#include <zephyr/dt-bindings/pwm/pwm.h>
```

Comment:
Shouldn't we reference it there instead?
https://github.com/catie-aq/zephyr_6tron-connector/blob/b4fb8215ce74d04cb19d30bcbc05144518f24f77/dts/bindings/sixtron-bus.yaml#L50